xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-theme/xwiki-platform-flamingo-theme-ui/src/main/resources/FlamingoThemes/Iceberg.xml

```diff
@@ -968,6 +968,12 @@
       <lessCode>/* Administration: Vertical Menus */
 @list-group-active-border: @list-group-border;
 
+/* Increasing default contrast for accessibility. */
+/* Slight change on @gray-light to make it pass the 4.5 contrast threshold on a white background. */
+@gray-light: #727272;
+/* Gray-light is too light on a light backgrounds. @gray for muted text is easier to read. */
+@text-muted: @gray;
```

Comment:
ok now I understand (you're doing an override) but I don't get why this is done in the Iceberg theme and not in the default css file.

Reply:
For some other themes it might not be a solution to get this value.
Typically we won't do the same changes for "dark" themes. We can't really solve all themes at once and I think it might create just as many issues in other themes if we change the default value.